### DISCUSSION TITLE
Include more informations in JWT claims

### DISCUSSION THREAD

**Discussion by Razikus** [supabase/supabase#14559] [Questions] [answered]
Hello

What i want to achieve is to include in JWT team identifier of user, so it would allow to make postgres policy same as
```auth.uid = table.user_id``` for some other fields

Is that possible? I couldn't find this in documentation

**Comment by GaryAustin1** [comment, marked as answer]
This should give you some ideas.  https://github.com/supabase-community/supabase-custom-claims
